php/php.editor/tools/ASTPHP5Parser.cup

```diff
@@ -2535,74 +2537,79 @@ non_empty_function_call_parameter_list:paramsList possible_comma
 ;
 
 non_empty_function_call_parameter_list ::=
-is_variadic:isVariadic expr_without_variable:var
+argument:arg
 {:
     List paramsList = new LinkedList();
-    Expression param = var;
-        if (isVariadic) {
-            param = new Variadic(isVariadicleft, varright, var);
-        }
-    paramsList.add(param);
+    paramsList.add(arg);
     RESULT = paramsList;
 :}
 
-| is_variadic:isVariadic variable:var
+| non_empty_function_call_parameter_list:paramsList T_COMMA argument:arg
 {:
-    List paramsList = new LinkedList();
-    Expression param = var;
-        if (isVariadic) {
-            param = new Variadic(isVariadicleft, varright, var);
-        }
-    paramsList.add(param);
+    paramsList.add(arg);
     RESULT = paramsList;
 :}
+;
 
-| T_REFERENCE:start w_variable:var
+argument ::=
+argument_expr:var
 {:
-    List paramsList = new LinkedList();
-    Expression var_ref = new Reference(startleft, varright, var);
-    paramsList.add(var_ref);
-    RESULT = paramsList;
+    RESULT = var;
 :}
 
-| error:expr
+| T_ELLIPSIS:e argument_expr:var
```

Comment:
If we use `is_variadic`, it conflicts with `function_call_parameter_list`.

Reply:
OK